backend/models/dtos/stats_dto.py

```diff
@@ -141,3 +141,19 @@ def __init__(self):
     # avg_completion_time = IntType(serialized_name='averageCompletionTime')
     organisations = ListType(ModelType(OrganizationListStatsDTO))
     campaigns = ListType(ModelType(CampaignStatsDTO))
+
+
+class TaskStatsDTO(Model):
+    """ DTO for tasks stats for a single day """
+
+    date = DateType(required=True)
+    mapped = IntType(serialized_name="mapped")
+    validated = IntType(serialized_name="validated")
+    invalidated = IntType(serialized_name="invalidated")
+    bad_imagery = IntType(serialized_name="badImagery")
+
+
+class AllTaskStatsDTO(Model):
+    """ Contains all tasks stats broken down by day"""
+
+    stats = ListType(ModelType(TaskStatsDTO))
```

Comment:
An improvement we could do - initializing `stats` variable in constructor, similar to other DTO classes. That way we ensure objects are initialized properly. From the execution perspective, either ways will work, and it's a matter of style, but to maintain uniformity in the code, I suggest we user constructors for initialization.